### ISSUE TITLE
Wrong appearance of non-ASCII characters in profile password

### ISSUE THREAD

**Issue by redmanmale** [uTox/uTox#826]
I know it's a bad idea to use non-ASCII characters in passwords, but still.

**Steps to reproduce**
* new profile
* go to profile password
* type some non-ASCII characters into password field

**Expected behaviour**
One asterisk for each character.

**Actual behaviour**
_Two_ asterisks for each character.

**Notes**
Encryption works just fine, only appearance is wrong.

**Env**
Win 7 x64, x86 + Ubuntu 16.04 x64
uTox v.0.14.0 + develop (b07c290)

**Comment by GrayHatter** [comment]
> I know it's a bad idea to use non-ASCII characters in passwords, but still.

No, it's not... and if uTox is breaking because of it, that's a huge issue that we need to fix!

**Comment by redmanmale** [comment]
Length is invalid when utf-8 characters are used.
https://github.com/uTox/uTox/blob/develop/src/ui/edit.c#L117

**Comment by dkmoz** [comment]
The problem is that edit->length is the text length in bytes. This could be fixed if we had two fields in struct edit: one for byte count and one for character count. Is there a better way to do this?

**Comment by redmanmale** [comment]
Or we could detect either string contains non-ASCII characters or not, and if yes, then use length/2.

**Comment by dkmoz** [comment]
UTF-8 symbols can take up to 4 bytes of space so dividing it by 2 won't work

**Comment by redmanmale** [comment]
I took a closer look and to fix this we have to rewrite almost whole `edit.c`
Maybe it's just not worth the trouble.
